s3Helper.js

```diff
@@ -2,9 +2,9 @@ const AWS = require('aws-sdk');
 const s3 = new AWS.S3();
 
 const config = require('./config');
-AWS.config.loadFromPath('./prod-s3.json');
 
 function writeToS3(agency, currentTime, data, isRaw) {
+  AWS.config.loadFromPath('./prod-s3.json');
```

Comment:
@EddyIonescu hopefully this won't affect prod. For people without the s3 creds this will get imported even when writing locally to cassandra since this and s3Helper are both globally imported.